modules/mono/editor/GodotTools/GodotTools/Utils/OS.cs

```diff
@@ -39,7 +39,7 @@ public static class Platforms
             public const string Windows = "windows";
             public const string MacOS = "macos";
             public const string LinuxBSD = "linuxbsd";
-            public const string Android = "android";
+            public const string Android = "linux-bionic";
```

Comment:
These are the Godot platform identifiers, not the .NET RIDs, those are in the `DotNetOS` static class which is just below this one.

Reply:
> These are the Godot platform identifiers, not the .NET RIDs, those are in the `DotNetOS` static class which is just below this one.

I'm sorry